src/local-editor-service.ts

```diff
@@ -94,24 +102,35 @@ export class LocalEditorService extends EditorService {
   async getReferencesForFeatureAtPosition(
       localPath: string,
       position: SourcePosition): Promise<SourceRange[]|undefined> {
-    const document = await this._analyzer.analyze(localPath);
+    const analysis = await this._analyzer.analyze([localPath]);
+    const document = analysis.getDocument(localPath);
+    if (!(document instanceof Document)) {
```

Comment:
If it's not a document then it's a Warning, so it'll be sent over when the editor asks for warnings.